tools/CleanRebuild.cmd

```diff
@@ -1,8 +1,8 @@
 @REM this script cleans all of the artifact locations and rebuilds the project
-call cd ..
```

Comment:
I modified the script so it doesn't need to changed directory, and will work correctly regardless of what the current directory is.